benchmark/net/chunky_http_client.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
```

Comment:
Style: the convention is to use single quotes.  There is usually a blank line after 'use strict'.